src/vault.py

```diff
@@ -126,6 +126,16 @@ def enable_audit_device(self, device_type: str, path: str) -> None:
         )
         logger.info("Enabled audit device %s", device_type)
 
+    def create_snapshot(self) -> requests.Response:
+        """Create a snapshot of the Vault data."""
+        logger.info("Created snapshot of the raft backend")
```

Comment:
Please log after the action was completed, here it is possible for the `created snapshot ...` log to display and the snapshot not to have been done. This applies in other places too

Reply:
Logging improved.